app/views/families/_families_table_header.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="table">
+  <div class="tr th">
+      <div class="td"><%= t ".application.families.show.last_name" %></div>
```

Comment:
Parens?

Reply:
Also, usually I use here `Family.human_attribute_name(:last_name)`

Ref: 
- https://stackoverflow.com/questions/4786733/human-attribute-names-in-rails-3-x#7593892
- https://apidock.com/rails/ActiveRecord/Base/human_attribute_name/class

Reply:
Nice to know! I used it, thanks!